Source/WebCore/dom/Element.cpp

```diff
@@ -150,6 +150,7 @@
 #include "WebAnimation.h"
 #include "WebAnimationTypes.h"
 #include "WheelEvent.h"
+#include "WritingSuggestionData.h"
```

Comment:
Why do we need to add this include? I don’t see any new code in this file. This must be left over from an earlier version of this patch.